src/plone/restapi/services/facet_count/get.py

```diff
@@ -0,0 +1,54 @@
+from plone.restapi.deserializer import json_body
+from plone.restapi.services import Service
+from plone.restapi.services.querystringsearch.get import QuerystringSearch
+from Products.CMFCore.utils import getToolByName
+import json
+
+
+class FacetCountGet(Service):
+    """Returns facet count."""
+
+    def reply(self):
+        facet_count = {}
+        ctool = getToolByName(self.context, "portal_catalog")
+        body = json_body(self.request)
+
+        facet = body.get("facet", None)
+        query = body.get("query", None)
+
+        try:
+            index = ctool._catalog.getIndex(facet)
+        except:
+            index = None
+
+        if query:
+            body["query"] = [qs for qs in query if qs["i"] != facet]
+            self.request.set("BODY", json.dumps(body))
+
+        brains = QuerystringSearch(self.context, self.request).getResults()
+
+        brains_rids = set(brain.getRID() for brain in brains)
+        index_rids = [rid for rid in index.documentToKeyMap()] if index else []
```

Comment:
Yes, we can make it return the count for multiple facets but we would still have to make a querystringsearch for each facet because we have to exclude the facet itself from the query.

Reply:
@razvanMiu oh right, I skipped over that part. Makes sense. Then there's probably less benefit from combining into a single request for multiple facets.